include/boost/geometry/util/precise_math.hpp

```diff
@@ -457,7 +457,10 @@ RealNumber incircle(std::array<RealNumber, 2> const& p1,
     RealNumber det = A_13 * (A_21_x_A_32[0] - A_31_x_A_22[0])
       + A_23 * (A_31_x_A_12[0] - A_11_x_A_32[0])
       + A_33 * (A_11_x_A_22[0] - A_21_x_A_12[0]);
-    if(Robustness == 0) return det;
+    if (BOOST_GEOMETRY_CONDITION(Robustness == 0))
```

Comment:
Can this (and similar) made `CONSTEXPR` as well? Apart from needing the `else` branch, but you did that elsewhere too